docs/modules/configuration/pages/secrets-management.adoc

```diff
@@ -169,3 +169,5 @@ image::vault.png[Secrets in Vault,width=70%]
 db.connection.test.username=HASHI_CORP_VAULT(secret/vividus/test/username)
 db.connection.test.password=HASHI_CORP_VAULT(secret/vividus/test/password)
 ----
+
+=== xref:ROOT:plugin-azure-resource-manager.adoc#_secrets_management_tools[Azure Key Vault]
```

Comment:
add basic description, like "AKV is supported, but it requires plugin, check details here"